testing/main.tf

```diff
@@ -0,0 +1,111 @@
+provider "aws" {
+  region                      = var.aws_region
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+###
+### SECURITY GROUPS
+###
+resource "aws_security_group" "allow-all" {
+  name   = "${var.tf_user}-allow-all-${var.GITHUB_RUN_ID}"
+  vpc_id = "vpc-07402b459d3b18976"
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name       = "allow-all"
+    Owner      = var.tf_user
+    github_run = "${var.GITHUB_RUN_ID}"
+  }
+}
+
+###
+### EC2 INSTANCES
+###
+# EC2 Instance for RKE2 Cluster - Manager
+resource "aws_instance" "control_node" {
+  count = var.control_nodes
+
+  ami           = lookup(var.amis[var.aws_region], var.os)
+  instance_type = var.instance_type
+  subnet_id     = "subnet-0523d8467cf8e5cec"
```

Comment:
Shouldn't this be a var?

Reply:
doh. I created the variable already, but didn't set it. Good catch.